synapse/lib/node.py

```diff
@@ -424,3 +430,19 @@ def delete(self, force=False):
         self.snap.buidcache.pop(self.buid)
 
         self.form.wasDeleted(self)
+
+class Path:
```

Comment:
What does this have to do with nodes?

Reply:
This tracks the progress of nodes passing through the storm runtime ( and can, for example, be used to record per-node variables and record pivot paths traversed to produce a given node )